package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.4.4",
   "author": "Aragon Association <legal@aragon.org>",
   "license": "AGPL-3.0",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
```

Comment:
Ah nice, is it still expected to use `main` as the commonJS export in package.jsons? I remember there might've been another key being proposed, but haven't really haven't followed any discussion.

Reply:
Sounds good!